pkg/kubelet/types/constants.go

```diff
@@ -20,3 +20,13 @@ const (
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 )
+
+// DefaultSysctlWhitelist returns the default whitelist of sysctls and sysctl patterns (ending in *).
+func DefaultSysctlWhitelist() []string {
+	return []string{
+		"kernel.shm*",
```

Comment:
I have removed it now from the whitelist, but left it in the greylist. So users can manually enable shmmni if needed.